encrypted-media/util/drm-messagehandler.js

```diff
@@ -0,0 +1,230 @@
+// Expect utf8decoder and utf8decoder to be TextEncoder('utf-8') and TextDecoder('utf-8') respectively
+//
+// drmconfig format:
+// { <keysystem> : {    "serverURL"             : <the url for the server>,
+//                      "httpRequestHeaders"    : <map of HTTP request headers>,
+//                      "servertype"            : "microsoft" | "drmtoday",                 // affects how request parameters are formed
+//                      "certificate"           : <base64 encoded server certificate> } }
+//
+
+drmconfig = {
+    "com.widevine.alpha": {
+        "serverURL": "https://lic.staging.drmtoday.com/license-proxy-widevine/cenc/",
+        "servertype" : "drmtoday",
+        "userId" : "12345",
+        "merchant" : "cablelabs"
+    },
```

Comment:
I'm not sure where to start to get that certificate, so help would be appreciated!

PlayReady requires a certificate to be used for `persistent-usage-record` sessions. I am not sure about `persistent-license` yet.

Does Widevine _require_ a certificate for any specific use-cases ? If not, what do you think a `setServerCertificate` test would look like ?